BUILD.md

```diff
@@ -320,7 +320,7 @@ To build and run the Rust proxy:
 ```bash
 cargo build -p conduit-proxy
 CONDUIT_PROXY_LOG=trace \
-  CONDUIT_PROXY_PUBLIC_LISTENER=tcp://0:5432 \
+  CONDUIT_PROXY_PUBLIC_LISTENER=tcp://127.0.0.1:5432 \
```

Comment:
Thanks for the correction!

This should probably be `0.0.0.0` -- we want the public listener to listen on all interfaces, not just localhost.

Practically, this doesn't matter much in development environments, but it helps to signal the intent.

Reply:
Ahh of course. Fixed!